src/clr-angular/data/datagrid/render/table-renderer.spec.ts

```diff
@@ -51,6 +51,11 @@ export default function(): void {
       expect(body.textContent).not.toMatch('Hello');
       expect(body.textContent).toMatch('World');
     });
+
+    it('adds a11y roles', function() {
+      const rowContent = context.clarityElement.querySelector('.datagrid-body');
+      expect(rowContent.attributes.role.value).toEqual('rowgroup');
```

Comment:
`.toEqual` works fine, but it should be used for objects. Here, we are checking a string value, which is primitive so `.toBe` is more appropriate, I think.

Reply:
will update the tests. thanks!